library/kotlin/src/io/envoyproxy/envoymobile/ResponseHeaders.kt

```diff
@@ -0,0 +1,18 @@
+package io.envoyproxy.envoymobile
+
+/*
+ * Headers representing an inbound response.
+ */
+class ResponseHeaders: Headers {
+  /**
+   * Internal constructor used by builders.
+   *
+   * @param headers: Headers to set.
+   */
+  internal constructor(headers: Map<String, List<String>>) : super(headers)
+
+  /**
+   * HTTP status code received with the response.
+   */
+  val httpStatus = value(":status")?.first()?.toInt()
```

Comment:
I think we should probably avoid throwing an exception here if possible to match Swift